.github/workflows/test-pypi-packages.yml

```diff
@@ -0,0 +1,42 @@
+name: test pipy packages
```

Comment:
very cool you made this also.

Reply:
Without tests, no one will believe it works. Even me :)